examples/react/src/components/Balance.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from "@emotion/styled";
+import { useNwcRequester } from "@uma-sdk/uma-auth-client";
+import { useEffect, useState } from "react";
+
+const Balance = () => {
+  const [balance, setBalance] = useState(0);
+  const [isFetching, setIsFetching] = useState(true);
+  const { nwcRequester } = useNwcRequester();
+  const fetchBalance = async () => {
+    // TODO: Should also check for an invalid token.
+    if (!nwcRequester) {
+      return;
+    }
+    setIsFetching(true);
+    const response = await nwcRequester.getBalance();
```

Comment:
probably wanna try catch this but can do later

Reply:
Done. Should do elsewhere, but I can do that as a follow-up.